### DISCUSSION TITLE
Taxes reports integration

### DISCUSSION THREAD

**Discussion by musinit** [btcpayserver/btcpayserver#4290] [Plugins & integrations]
Hello! I love what you do. Open Source is the key for any platform to be Audited, adopted, and trusted.
What do you think about taxes reports? We can add integration with some Open Source tax calculation platforms so that people can generate taxes through their crypto transactions.

**Comment by Kukks** [comment]
Sure, do you have integrations in mind?

**Reply by pavlenex** [reply]
Sounds like a good idea for sure!

**Reply by musinit** [reply]
There are several options, the most popular taxes tracking tool are:
[cointracker](https://www.cointracker.io/)
[koinly](https://koinly.io/)
[US](https://turbotax.intuit.com/)

[taxcry](https://taxcry.io/)

I can help with integration for taxcry, because I am responsible for it. We can schedule a call and discuss details.

**Reply by dennisreimann** [reply]
> It works for Ethereum tokens only by now.

BTCPay Server doesn't support Ethereum anymore. In general, Altcoins and their integration aren't supported by us — the individual communities take care of that.

Do you have an idea how a general export could look like or what we as BTCPay Server could do to help users with their tax reports?

**Reply by musinit** [reply]
We can and will connect more networks.
I need some time to research, maybe I could suggest some solution for you

**Reply by musinit** [reply]
Can someone from maintainers pick up the time for a short call so we can discuss technical solution?
https://calendly.com/d/g6q-h5b-x2d/btcpay-taxcry-integration
